src/App.tsx

```diff
@@ -1,61 +1,121 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+const getUserById = (userId: number) => {
+  return usersFromServer.find(user => user.id === userId) || null;
+};
+
+const listTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+const getNewPostId = (todos: Todo[]) => {
+  const maxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
There is a syntax error in the `getNewPostId` function. The spread operator is incorrectly used. Ensure that you are correctly mapping over the todos to find the maximum ID.